stock_picking_partner_language/picking_report.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <report 
```

Comment:
Is it needed to overwrite full report definition? I think you can use normal XML record definition for overwriting only affected fields (name and file)